airbyte-webapp/src/packages/cloud/App.tsx

```diff
@@ -80,9 +81,11 @@ const App: React.FC = () => {
         <I18NProvider>
           <StoreProvider>
             <Suspense fallback={<LoadingPage />}>
-              <Services>
-                <Routing />
-              </Services>
+              <Router>
```

Comment:
@Jamakase Do you recall any specific reason, why the `Router` was inside the `Routing` component for the cloud package, but on this level for the OSS part? I've moved it now also into the App component for cloud (so the `useLocation` hook will be usable within routing) and everything seems to work, but maybe you recall any specific reason why this wouldn't work?

Reply:
Not sure why it was done that way. We could definitely move it upper in the tree.